lab_1_classify_by_unigrams/main.py

```diff
@@ -14,8 +14,8 @@ def tokenize(text: str) -> list[str] | None:
     """
     if not isinstance(text, str):
         return None
-
-    return [token.lower() for token in text if token.isalpha()]
+    punc = ''''1234567890!"#$%&'()’*º+,-./:;<=>?@[\\]^_\' \'`{\\\n|}~'''
```

Comment:
It is strange changes... need to be revert to main

Reply:
I just wanted to keep my own lab1 implimentation and not use Anna's, that is what Git wanted me to do (because her code is the best etc), so I just kept my own code

Reply:
ok, create a separate branch for it. But in main is must be reverted